docs/README-AWS.md

```diff
@@ -7,10 +7,11 @@ The Elastic Serverless Forwarder is an AWS Lambda function that ships logs from
 - Direct SQS message payload input
 - S3 SQS Event Notifications input
 - Kinesis Data Stream input
+- Cloudwatch logs events input
 
 The config yaml file (details described below) acts as an input where the user, based on input type, configures things like SQS queue ARN and Elasticsearch connection information. Multiple input sections can be created in the configuration file pointing to different queues that match specific log types.
 
-A continuing SQS queue is set up by the Lambda deployment automatically. It is used to trigger a new function invocation so that Lambda can continue from exactly where the last function run was terminated. By default a Lambda function runs for a max of 15 minutes. When processing large log files there’s a possibility that the function may be exited by AWS in the middle of processing a log file. The code handles this scenario gracefully by keeping track of the file and offset its processing.
+A continuing SQS queue is set up by the Lambda deployment automatically. It is used to trigger a new function invocation so that Lambda can continue from exactly where the last function run was terminated. By default a Lambda function runs for a max of 15 minutes. When processing large log files, sqs drect message payload or cloudwatch logs events there’s a possibility that the function may be exited by AWS in the middle of processing a log file. The code handles this scenario gracefully by keeping track of the last offset it processed.
```

Comment:
remove "a log file" as it doesn't make sense for other input types.
Fix typo drect to direct.